src/main.js

```diff
@@ -3,6 +3,22 @@ import { createDropdownTypes, createPokemonList } from './dom/index.js';
 
 const addListeners = () => {
 	// Code goes here
+	
+	//Theme change
+	const themeSwitch = document.getElementById('header-checkbox');
+	themeSwitch.addEventListener('change', changeTheme);
+
+	function changeTheme(event) {
+		//var status = event.target.checked;
+		var mainContent = document.querySelector(".main-content");
+		var navBar = document.querySelector(".header__nav");
```

Comment:
You only need to `toggle` the class in the `mainContent` element. In the CSS, there is a declaration that changes the background color of the header.
```
.dark .header__nav {
	background-color: #16161d;
}
```